website/static/js/projectSettings.js

```diff
@@ -107,7 +107,40 @@ var ProjectSettings = oop.extend(
                     }
                 }
             };
-        }
+        },
+        enableMailingList: function() {
+            var request = $.ajax({
+                type: 'post',
+                dataType: 'json',
+                url: nodeApiUrl + 'mailing_list/'
+            });
+            request.done(function() {
+                window.location.reload();
+            });
+            request.fail($osf.handleJSONError);
+        },
+        disableMailingList: function(nodeType) {
+
+            var message = '<p>Are you sure that you want to disable the mailing list for this ' + nodeType + '?';
```

Comment:
Yeah, what this looks like it should be doing is actually handled by `mailingListTreebeard.js`, this is left over from before that was created. I'll remove it